main.go

```diff
@@ -176,6 +176,12 @@ func main2() error {
 				if len(response.Value.Content) == 1 {
 					if body := response.Value.Content.Get("application/json"); body != nil {
 						resp = schemaRefParse(body.Schema, "")
+						// If the response is not a referenced type and is instead defined inline,
+						// we need to build the response type.
```

Comment:
theoretically, i guess you could use an inline Go type, but this will do for now, I guess (there's still the possibility of a clash).